pkg/operator/status/inertia.go

```diff
@@ -0,0 +1,54 @@
+package status
+
+import (
+	"fmt"
+	"regexp"
+	"time"
+)
+
+// Inertia returns the inertial duration for the given condition type.
+type Inertia func(conditionType string) time.Duration
+
+// InertiaCondition configures an inertia duration for a given set of
+// condition types.
+type InertiaCondition struct {
+	// ConditionType is a regular expression selecting condition types
+	// with which this InertiaCondition is associated.
+	ConditionType *regexp.Regexp
```

Comment:
nit: `ConditionTypeMatcher` or something similar to highlight this is a regexp ?

Reply:
> nit: `ConditionTypeMatcher` or something similar to highlight this is a regexp ?

I think `Regexp`-ness is already clear from the property definition here, and Go will throw useful compiler errors for folks who think it's a string or whatever.  But I don't care all that much; rerolled to use your name with 1a1079fd5 -> 72d73fac0.

Also in that reroll, I've changed:

```go
type Inertia func(conditionType string) time.Duration
```

to:

```go
type Inertia func(conditionType string) time.Duration
```

in case folks want to write `Inertia` implementations that switch on the `Reason` or some such.